File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int count = (n + 1) / 2;
        int[] answer = new int[count];
        
        for (int i = 0, num = 1; i < count; i++, num += 2) {
            answer[i] = num;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.cs
using System;

public class Solution {
    public int[] solution(int n) {
        int count = 0;
        for(int i = 1; i <= n; i+=2){
            count++;
        }
        int[] answer = new int[count];
        count = 0;
        for(int i = 1; i <= n; i+=2){
            answer[count++] = i;
        }
        return answer;
    }
}